packages/stacking/README.md

```diff
@@ -0,0 +1,94 @@
+# @stacks/stacking [![npm](https://img.shields.io/npm/v/@stacks/stacking?color=red)](https://www.npmjs.com/package/@stacks/stacking)
```

Comment:
It looks like this readme doesn't cover all available methods. I found this in the tutorial:
- stackingEnabledNextCycle
- getCycleDuration
- secondsUntilNextCycle
- hasMinimumRequiredStxAmount

I didn't check if there are more but I'd suggest we have them all in this readme (so it can act as a reference)